cypress/support/pages/contactForm.pageObject.js

```diff
@@ -19,11 +19,11 @@ class ContactFormPageObject extends PageObject {
     return cy.contains('.btn', 'Send message');
   }
 
-  typeEmail(email) {
+  typeEmail() {
     this.emailField.type(email, { force: true });
```

Comment:
The 'typeEmail' method is missing a parameter for 'email'. Add 'email' as a parameter to this method and use it in the 'type' function to avoid reference errors.